### ISSUE TITLE
[To be reproduced] Etherlime solidity-coverage breaks the etherlime-cli

### ISSUE THREAD

**Issue by vongohren** [LimeChain/etherlime#70]
Im getting version 0.9.5

<img width="1044" alt="screen shot 2018-09-09 at 11 37 24" src="https://user-images.githubusercontent.com/1012055/45263184-ddee3b80-b424-11e8-9d0d-5171d90b6ce3.png">

**Comment by Perseverance** [comment]
Thanks for opening this issue. Couple of questions:
- did ‘npm i -g etherlime’ ran successfully 
- what kind of os do you use and what versions your node and npm are?

**Comment by ochikov** [comment]
Hello @vongohren ,
I did a few tests with your provided version of node and npm. Everything is building fine. I am posting the logs. Do you think about something different in our configurations? Could you provide us with a repo, so we can take further action to solve your problem?

```MacBook-Pro-4:OgiTestDelete ognyanchikov$ npm i -g etherlime
/usr/local/bin/etherlime -> /usr/local/lib/node_modules/etherlime/cli.js

> sha3@1.2.2 install /usr/local/lib/node_modules/etherlime/node_modules/sha3
> node-gyp rebuild

  CXX(target) Release/obj.target/sha3/src/addon.o
  CXX(target) Release/obj.target/sha3/src/displayIntermediateValues.o
  CXX(target) Release/obj.target/sha3/src/KeccakF-1600-reference.o
  CXX(target) Release/obj.target/sha3/src/KeccakNISTInterface.o
  CXX(target) Release/obj.target/sha3/src/KeccakSponge.o
  SOLINK_MODULE(target) Release/sha3.node
+ etherlime@0.9.5
added 288 packages from 991 contributors and updated 1 package in 14.905s
MacBook-Pro-4:OgiTestDelete ognyanchikov$ etherlime init 
===== Installing etherlime =====
===== Generating package.json =====

> sha3@1.2.2 install /Users/ognyanchikov/OgiTestDelete/node_modules/sha3
> node-gyp rebuild

  CXX(target) Release/obj.target/sha3/src/addon.o
  CXX(target) Release/obj.target/sha3/src/displayIntermediateValues.o
  CXX(target) Release/obj.target/sha3/src/KeccakF-1600-reference.o
  CXX(target) Release/obj.target/sha3/src/KeccakNISTInterface.o
  CXX(target) Release/obj.target/sha3/src/KeccakSponge.o
  SOLINK_MODULE(target) Release/sha3.node
+ etherlime@0.9.5
added 289 packages from 992 contributors and audited 550 packages in 20.427s
found 0 vulnerabilities


===== Creating contracts file structure =====
===== Creating tests file structure =====
===== Creating deployment file structure =====
Etherlime was successfully initialized! Check ./deployment/deploy.js for your deployment script.
MacBook-Pro-4:OgiTestDelete ognyanchikov$ code .
MacBook-Pro-4:OgiTestDelete ognyanchikov$ etherlime ganache
[0] Address: 0xd9995bae12fee327256ffec1e3184d492bd94c31 Private key: 0x7ab741b57e8d94dd7e1a29055646bafde7010f38a900f55bbd7647880faa6ee8
[1] Address: 0xd4fa489eacc52ba59438993f37be9fcc20090e39 Private key: 0x2030b463177db2da82908ef90fa55ddfcef56e8183caf60db464bc398e736e6f
[2] Address: 0x760bf27cd45036a6c486802d30b5d90cffbe31fe Private key: 0x62ecd49c4ccb41a70ad46532aed63cf815de15864bc415c87d507afd6a5e8da2
[3] Address: 0x56a32fff5e5a8b40d6a21538579fb8922df5258c Private key: 0xf473040b1a83739a9c7cc1f5719fab0f5bf178f83314d98557c58aae1910e03a
[4] Address: 0xfec44e15328b7d1d8885a8226b0858964358f1d6 Private key: 0x823d590ed2cb5e8493bb0efc834771c1cde36f9fc49b9fe3620ebd0754ad6ea2
[5] Address: 0xda8a06f1c910cab18ad187be1faa2b8606c2ec86 Private key: 0xd6d710943471e4c37ceb787857e7a2b41ca57f9cb4307ee9a9b21436a8e709c3
[6] Address: 0x8199de05654e9afa5c081bce38f140082c9a7733 Private key: 0x187bb12e927c1652377405f81d93ce948a593f7d66cfba383ee761858b05921a
[7] Address: 0x28bf45680ca598708e5cdacc1414fcac04a3f1ed Private key: 0xf41486fdb04505e7966c8720a353ed92ce0d6830f8a5e915fbde735106a06d25
[8] Address: 0xf0508f89e26bd6b00f66a9d467678c7ed16a3c5a Private key: 0x6ca40ba4cca775643398385022264c0c414da1abd21d08d9e7136796a520a543
[9] Address: 0x87e0ed760fb316eeb94bd9cf23d1d2be87ace3d8 Private key: 0xfac0bc9325ad342033afe956e83f0bf8f1e863c1c3e956bc75d66961fe4cd186

Listening on http://localhost:8545
```

**Comment by vongohren** [comment]
Hey I think I found the entrance point for the errors.
So what I tried to do was to have etherlime as a project dependency and using npx to run it. Also call it directly from node_modules/.bin.
I also tried to use yarn add global to check if that made any change.
But all these things fail with the same error.

But npm install -g work for some reason.

Maybe it is lacking some kind of dependecy that I by a chance have in my npm global space, but not in my yarn global space or on the project specifics
